modules/static/cncf/xds/buf.md

```diff
@@ -11,4 +11,4 @@ deps:
   - buf.build/cncf/xds:<GIT_COMMIT_TAG>
 ```
 
-For more information, see the [documentation](https://docs.buf.build/bsr/overview).
+For more information, see the [documentation](https://buf.build/docs/bsr/overview).
```

Comment:
This redirects me to https://buf.build/docs/bsr/introduction, show we update it so it's the non-redirected path?

Reply:
good point. sorry- i merged before this comment came in. but, i feel like getting rid of the subdomain was more important than the redirect.